README.md

```diff
@@ -28,7 +28,7 @@ A simple template is shown below which contains some example headers for use wit
 
 <b>Before you continue, please check the Pre-Requisites above (for example, that you have a valid GitHub.com account)</b>
 
-We are using [Netlify](https://www.netlify.com/) as the backend CRM and all content will be added here.  No technical knowledge is required as the interface is fairly WYSIWYG.
+We are using [Netlify](https://www.netlify.com/) as the backend CMS and all content will be added here.  No technical knowledge is required as the interface is fairly WYSIWYG.
```

Comment:
actually we are using github as the CMS, we are just using the Netlify for hosting and we are using their front end editor for editing files.

Reply:
Ah gotcha, just wanted to avoid confusion since CRM generally stands for customer relationship management.

Reply:
Thank you for catching this!!

For more info, see https://www.netlifycms.org/